submissions/HTML/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+
+<body>
+
+<div class="topnav">
+  <a class="active" href="#home">Home</a>
+  <a href="#news">News</a>
+  <a href="#contact">Contact</a>
+  <a href="#about">About</a>
+</div>
```

Comment:
The entire file isnt properly formatted. This div is under the body tag, they shouldnt start at the same point. Kindly check the samples in the lesson video/slides.